caracal.js

```diff
@@ -49,7 +49,7 @@ app.use('/img/IIP/raw/', iipHandler);
 
 // loader, proxy
 app.use('/loader/', auth.loginHandler(auth.PUBKEY));
-app.use('/loader/', permissionHandler(['Admin', 'Editor']));
+app.use('/loader/', permissionHandler(['Admin']));
```

Comment:
This is the change I'm referring to.

Reply:
Oh, my bad... I'm sorry, it's got to be a development thing that I was testing and forgot to change back. I'll edit it back to admin and editor.